packages/manager/modules/telecom-spare/src/modems/order/modems-order.controller.js

```diff
@@ -0,0 +1,127 @@
+import _ from 'lodash';
```

Comment:
You should only import the functions you need e.g : 
```suggestion
import get from 'lodash/get';
```